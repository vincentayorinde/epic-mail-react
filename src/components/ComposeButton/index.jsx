
import React from 'react';

const ComposeButton = ({btnValue}) => (
  <div className="box-1-inbox">
    <button> <i className="fas fa-pen-square"></i><a href="compose.html">{btnValue}</a></button>
  </div>
)

export default ComposeButton;
 