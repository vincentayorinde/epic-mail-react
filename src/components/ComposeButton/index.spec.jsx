
import React from 'react';
import { shallow } from 'enzyme';
import ComposeButton from './index';

describe('ComposeButton Component', () => {
  it('Should render without errors', () => {
    const component = shallow(<ComposeButton />);
    expect(component.find('.box-1-inbox').length).toBe(1);
    expect(component).toMatchSnapshot();
  });
});
