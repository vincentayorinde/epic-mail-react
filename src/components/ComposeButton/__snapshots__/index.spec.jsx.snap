// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ComposeButton Component Should render without errors 1`] = `
<div
  className="box-1-inbox"
>
  <Link
    to="/compose"
  >
    <button>
       
      <i
        className="fas fa-pen-square"
      />
    </button>
  </Link>
</div>
`;
